frontend/translations/en.po

```diff
@@ -2069,6 +2069,30 @@ msgstr "Hide resolved comments"
 msgid "labels.import"
 msgstr "Import"
 
+#: src/app/main/ui/workspace/tokens/modals/import.cljs:116
+msgid "workspace.token.import-tokens"
```

Comment:
Should this be `workspace.tokens...` instead of `workspace.token`?

Reply:
All translate strings for tokens are called "workspace.token" so I will create a separated PR for that.